site/guide/monitoring/work-with-metrics-over-time.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "Work with metrics over time"
+date: last-modified
+---
+
+Once generated via the {{< var validmind.developer >}}, view and add metrics over time to your ongoing monitoring plans in the {{< var validmind.platform >}}.
+
+Metrics over time refers to the continued monitoring of a model's performance once it is deployed. Tracking how a model performs as new data is introduced or conditions change ensures that it remains accurate and reliable in real-world environments where data distributions or market conditions shift. 
+
+- Model performance is determined by continously measuring metrics and comparing them over time to detect degration, bias, or shifts in the model's output. 
```

Comment:
```suggestion
- Model performance is determined by continuously measuring metrics and comparing them over time to detect degradation, bias, or shifts in the model's output. 
```

Reply:
NO need to apologise! Thank you so much for reviewing, and I should have caught that dur.